tests/test_scalar_product.f90

```diff
@@ -32,7 +32,7 @@ program test_scalar_product
   character(len=20) :: BC_x(2), BC_y(2), BC_z(2)
 
   character(len=5), dimension(4), parameter :: test = &
-    ["DIR_X", "DIR_Y", "DIR_Z", "DIR_C"]
+                                           ["DIR_X", "DIR_Y", "DIR_Z", "DIR_C"]
```

Comment:
This will keep raising a warning when fprettify is run over tests but the check policy doesn't do fprettify over tests so no problem for the moment.

Reply:
This is what `fprettify` formatted it to though?

Reply:
Basically all the changes are to satisfy the `fprettify` test

Reply:
Thats one problem with fprettify. It tries to pad continuation lines so that they start at the right location, but if the line width goes above 79 with the correct amount of padding it ends up with this weird state, where it adds spaces only until the line width is 79... It gives a warning and expects user to fix by manually breaking the line into multiple lines.

Reply:
I've reformatted this to address fprettify issues.